pretyping/reductionops.ml

```diff
@@ -1284,7 +1296,7 @@ let local_whd_state_gen flags _env sigma =
       else s
 
     | Rel _ | Var _ | Sort _ | Prod _ | LetIn _ | Const _  | Ind _ | Proj _
-      | Int _ | Float _ -> s
+      | Int _ | Float _ | Array _ (* FIXME *) -> s
```

Comment:
This is the last bit that should be addressed, after #11707 and the following refactoring are merged.

Reply:
What needs doing here?

Reply:
You're right, nothing in fact, since primitives can't be reduced by these local reductions.